rpcs3/evdev_joystick_handler.cpp

```diff
@@ -632,7 +659,14 @@ int evdev_joystick_handler::add_device(const std::string& device, bool in_settin
 				close(fd);
 				continue;
 			}
-			const std::string name = et.name + ": " + libevdev_get_name(dev);
+
+			std::string name = get_device_name(dev);
+
+			if (unique_names.find(name) == unique_names.end())
```

Comment:
The name format is not the same for the first instance and the other ones AFAIU.

For consistency, I'd just use the same name format all the time:

    std::string name = get_device_name(dev);
    name = fmt::format("%d. %s", ++unique_names[name], name);

Reply:
That's possible. But what do others say.

Reply:
actually I think using numbers for all names is going to create inconsistencies. For example when you add different controllers on the next boot. You'd have to make sure the correct controllers always end up at the same numbers. This is not possible with evdev (I think). We get different events each Linux reboot.

Reply:
Well that's the whole thing: with `std::unordered_map<std::string, u32> unique_names;` the number are per device name. If you have two different devices they will be called "1. Ultimate Gamepad" and "1. Killer Gamepad" but if you have two same gamepads they will be called "1. Ultimate Gamepad" and "2. Ultimate Gamepad". Now the two game pads "Ultimate Gamepads" could be swapped between reboot but if they are really identical that's not a problem but it's actually a feature.

Reply:
you know... it's exactly what i do right now.
only i don't use 1.
did you even try?

Reply:
OK, I failed to parse you comment because I assumed you was not the author of the PR :)